crose72/OperationSquirrel
# Updated mav_cmd to use encode function rather than the pack function

Using the encode function to send command_long and command_int instead of the pack function.  Encode uses the pack function directly.  Also subscribed to the POSITION_LOCAL_NED mavlink message and added a print function for it.

How to validate that it works:

In WSL/WSL2:

1. Compile a Debug build
2. Execute ./squirreldefender
3. You should see messages printing across your terminal.  Specifically look for `Local Position NED` and it's outputs (this was newly added).
4. Drone should fly in a zig zag pattern and then return to the launch point as defined in `sim_flight_test_4_VelocityControl.h`, a hardcoded test flight.

With the Jetson:

1. Compile a Debug build
2. Execute ./squirreldefender
3. You should see messages printing across your terminal.  Specifically look for `Local Position NED` and it's outputs (this was newly added).
4. Drone should follow whatever commands jetson provides (follow is currently used).
